### PR TITLE
Replication of efficient policy learning paper experiments

### PR DESCRIPTION
**Pull request by swager**
We are revising the efficient policy learning paper, and have redone all experiments using `policytree`. This commit adds complete replication files to the repo.

### CODE REVIEW

**Review by erikcs — APPROVED** [review]
Should we update `experiments/README` too and remove the "check_replication_epl*" files in that folder?

File: experiments/athey_wager_2017/figure_2_make_plot.R [hunk 1]
```diff
@@ -0,0 +1,40 @@
+library(ggplot2)
```
Comment (erikcs) [inline experiments/athey_wager_2017/figure_2_make_plot.R]: Can remove this import

File: experiments/athey_wager_2017/table_3_make_table.R [hunk 2]
```diff
@@ -0,0 +1,24 @@
+set.seed(1234)
```
Comment (erikcs) [inline experiments/athey_wager_2017/table_3_make_table.R]: set.seed is not necessary here?

File: experiments/athey_wager_2017/table_3_run.R [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+        #abline(0, 1, col = 2, lwd = 4)
+        
+        W.cut = cut(W.all, W.breaks)
+        W.bucket.all = model.matrix( ~ W.cut + 0)
+        W.bucket = W.bucket.all[1:n.train,]
+        
+        XWgrid = expand.grid(1:n.train, 1:length(W.mids))
+        
+        BX.0 = cbind(10, generate.basis(rbind(X, X.test), 3))
+        #BX.0 = cbind(10, rbind(X[,1:2], X.test[,1:2]))
```
Comment (erikcs) [inline experiments/athey_wager_2017/table_3_run.R]: This script contains a few commented out lines, can these perhaps just be removed?

**Comment by swager** [comment]
Thanks @erikcs! And yes -- we should remove the `check_replication_EPL` files (I took those used them to build the replication files for FIG 2 here, so they're redundant now). And then, for the MAPL paper: We should also make a directory for the paper, with the expectation that we'll add complete replication files there based on `policytree` next time we revise that paper.

**Comment by erikcs** [comment]
Ok, if you merge this I can update the old files in `experiments` in a new PR!

**Pull request merged**